<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoSelectKey</class>
 <widget class="QDialog" name="autoSelectKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="ImageLabel" name="SinputVideo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>551</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
  <widget class="ImageLabel" name="SLabelsVideo">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>551</width>
     <height>331</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>labels video</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>571</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="videoPBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>581</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previousFrame">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/buttonback.png</normaloff>Resources/buttonback.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playAndPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/buttonstop2run.png</normaloff>Resources/buttonstop2run.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopPlay">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/buttonrun2pause.png</normaloff>Resources/buttonrun2pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextFrame">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/buttonadvance1.png</normaloff>Resources/buttonadvance1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>89</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="inVideoView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>474</x>
      <y>50</y>
      <width>83</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="frameIdx">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="inImgIdx">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="savePairs">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="PairSize">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>41</width>
      <height>0</height>
     </rect>
    </property>
    <property name="text">
     <string>PSize</string>
    </property>
   </widget>
   <widget class="QLabel" name="curLIdx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>46</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="curRIdx">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>61</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="isBackground1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>0</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Background</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/YuanQian/.designer/backup/Resources/check.png</normaloff>C:/Users/YuanQian/.designer/backup/Resources/check.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="NextOperation">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="StepReminde">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step Remide</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isSelectKeyFrameA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>81</width>
      <height>0</height>
     </rect>
    </property>
    <property name="text">
     <string>First</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/YuanQian/.designer/backup/Resources/check.png</normaloff>C:/Users/YuanQian/.designer/backup/Resources/check.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="isSelectKeyFrameB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>81</width>
      <height>0</height>
     </rect>
    </property>
    <property name="text">
     <string>Second</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/YuanQian/.designer/backup/Resources/check.png</normaloff>C:/Users/YuanQian/.designer/backup/Resources/check.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="FirstFrame">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="LeftCandIdx">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SecondFrame">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="RightCandId">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="isBackground">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QLabel" name="curBGIdx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>0</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bgIdx2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="VideoImgTranslation">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Translation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/YuanQian/.designer/backup/Resources/translation.png</normaloff>C:/Users/YuanQian/.designer/backup/Resources/translation.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>79</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header location="global">ImageShowWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
